package/lib-edgetpu/install.sh

```diff
@@ -0,0 +1,15 @@
+#! /bin/bash
+
+#
+# Copyright (c) 2020 cTuning foundation.
+# See CK COPYRIGHT.txt for copyright details.
+#
+# See CK LICENSE.txt for licensing details.
+#   
+# Installation script for the EdgeTPU library.
+#
+
+wget --no-cookies ${PACKAGE_URL}${PACKAGE_NAME}
```

Comment:
Please check for errors after each step e.g. with a [function](https://github.com/ctuning/ck-openvino/blob/master/package/lib-openvino/scripts.linux/install.sh#L14) like [here](https://github.com/ctuning/ck-openvino/blob/master/package/lib-openvino/scripts.linux/install.sh#L95).

Reply:
Ok. Done.